src/components/03-components/03-buttons/01-button/button.njk

```diff
@@ -0,0 +1,21 @@
+{{name}}{%- if (name == 'Outline-inverse') -%} 
+{% set bkg = "bg-base-darkest text-base-lightest " -%} 
```

Comment:
delete text-base-lightest -- unnecessary and removing it ensures that the inverse buttons get light text without hard coding it inn html